backend/app/services/channels/discord/user_resolver.py

```
@@ -0,0 +1,116 @@
+
+    def _resolve_selected_user(self, discord_user_id: int) -> Optional[User]:
+        target_user_id = self.user_mapping_config.get("target_user_id")
+        if not target_user_id:
+            logger.warning(
+                "[DiscordUserResolver] select_user mode: no target_user_id configured"
+            )
+            return None
+
+        user = (
+            self.db.query(User)
+            .filter(User.id == target_user_id, User.is_active == True)
```

Ruff (0.15.17)

[error] Avoid equality comparisons to `True`; use `User.is_active:` for truth checks

Replace with `User.is_active`

(E712)